plugins/content/pagenavigation/pagenavigation.php

```diff
@@ -89,7 +89,7 @@ public function onContentBeforeDisplay($context, &$row, &$params, $page = 0)
 				{
 					// Use created if modified is not set
 					case 'modified':
-						$orderby = $db->quoteName('a.modified') . ' IS NULL THEN ' .
+						$orderby = 'CASE WHEN ' . $db->quoteName('a.modified') . ' IS NULL THEN ' .
 						$db->quoteName('a.created') . ' ELSE ' . $db->quoteName('a.modified') . ' END';
```

Comment:
```suggestion
							$db->quoteName('a.created') . ' ELSE ' . $db->quoteName('a.modified') . ' END';
```
Make indentation consistent with the below case 'published' - This was not reported by Drone PHP CS check and was wrong also before this PR, but if you can, fix it here, too.